packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -97,8 +97,8 @@ export function ListViewItem(props) {
   let isSelected = state.selectionManager.isSelected(item.key);
   let showDragHandle = isDraggable && (isFocusVisibleWithin || isHovered || isPressed);
   let {visuallyHiddenProps} = useVisuallyHidden();
-  let isFirstItem = item.prevKey == null;
-  let isLastItem = item.nextKey == null;
+  let isFirstRow = item.prevKey == null;
```

Comment:
I thought ListView did items not rows?

Reply:
it uses `Item` but still follows the grid row/cell pattern. The above change was made just for code consistency/clarity